json-schema/Request-Context-v2.json

```diff
@@ -0,0 +1,267 @@
+            "system": {
+              "type": "string",
+              "description": "Url that identifies HPI Facility resource type",
+              "const": "https://standards.digital.health.nz/ns/hpi-facility-id"
+            },
+            "value": {
+              "type": "string",
+              "description": "HPI Facility Id in the form FXXNNN-C where X is alphanumeric, N is numeric and C is a check character",
+              "pattern": "^F[A-Z0-9]{2}[0-9]{3}-[A-Z]{1}$",
+              "example": "FZZ958-K"
+            }
+          },
+          "required": [ "use", "system", "value"]
+        },
+        "display": {
+          "type": "string",
+          "description": "The display name of the health facility", 
+          "example": "Medical Centre Flat-Unit"
+        }
+      },
+      "required": ["type","identifier","display"]
+    },
+    "hpiOrganisation": {
+      "type": "object",
+      "description": "Defines the health provider the user is currently working in, as an HPI Organisation FHIR Reference",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Fixed to refer to FHIR Organization resources",
+          "example": "Organization",
+          "pattern": "^Organization$"
+        },
+        "identifier": {
+          "type": "object",
+          "description": "These three attributes constitute a FHIR Identifier for an HPI Organisation resource",
+          "properties": {
+            "use": {
+              "type": "string",
+              "description": "Fixed to official",
+              "const": "official"
+            },
+            "system": {
+              "type": "string",
+              "description": "Url that identifies a HPI Organisation resource type",
+              "const": "https://standards.digital.health.nz/ns/hpi-organisation-id"
+            },
+            "value": {
+              "type": "string",
+              "description": "HPI Organisation Id in the form GXXNNN-C where X is alphanumeric, N is numeric and C is a check character",
+              "pattern": "^G[A-Z0-9]{2}[0-9]{3}-[A-Z]{1}$",
+              "example": "GZZ956-B"
+            }
+          },
+          "required": [ "use", "system", "value"]
+        },
+        "display": {
+          "type": "string",
+          "description": "The display name of the organisation", 
+          "example": "Flat-Unit Address Org"
+        }
+      },
+      "required": ["type","identifier","display"]
+    },
+    "secondaryIdentifier": {
+      "type": "object",
+      "properties": {
+          "use": {
+              "type": "string"
+          },
+          "system": {
+              "type": "string"
+          },
+          "value": {
+              "type": "string"
+          }
+      },
+      "required": [
+          "use",
+          "system",
+          "value"
+      ]
+    },
+    "purposeOfUse": {
+        "type": "array",
+        "items": { "$ref": "purposeOfUseTypes" }
```

Comment:
This won't resolve - needs to be `"items": { "$ref": "#/definitions/purposeOfUseTypes" }`